admin/metrics/models.py

```diff
@@ -0,0 +1,19 @@
+from django.db import models
+
+
+class OSFStatistic(models.Model):
+    users = models.IntegerField(verbose_name='OSF users', default=0)
+    delta_users = models.IntegerField(default=0)
+    unregistered_users = models.IntegerField(verbose_name='Unregistered users',
+                                             default=0)
+    projects = models.IntegerField(verbose_name='Number of projects',
+                                   default=0)
+    delta_projects = models.IntegerField(default=0)
+    public_projects = models.IntegerField(
+        verbose_name='Number of public projects', default=0)
+    delta_public_projects = models.IntegerField(default=0)
+    registered_projects = models.IntegerField(
+        verbose_name='Number of projects registered', default=0
+    )
+    delta_registered_projects = models.IntegerField(default=0)
+    date = models.DateTimeField(default=None)
```

Comment:
Kinda goofy that we have to duplicate this code with the [migration code](https://github.com/CenterForOpenScience/osf.io/blob/349e0e061372413688133b48562799addb6524b6/admin/metrics/migrations/0001_initial.py#L13-L28). Is there no way around that?